DFKPS/iocBoot/iocDFKPS-IOC-01/st-8800.cmd

```diff
@@ -0,0 +1,15 @@
+## Initialise the comms with the PSU
+asynOctetConnect("DFKINIT","L0")
+asynOctetWrite DFKINIT "ADR 000\r"
+
+## Load our record instances
+dbLoadRecords("$(TOP)/Db/DFKPS_8800_status.db", "device=$(MYPVPREFIX)$(IOCNAME), P=$(MYPVPREFIX)$(IOCNAME):, port=L0")
+
+##ISIS## Stuff that needs to be done after all records are loaded but before iocInit is called 
```

Comment:
From here to the end should be in st.cmd, as it's the same for each of the files